src/backend/lib/redis.js

```diff
@@ -14,7 +16,13 @@ const useMockRedis = process.env.MOCK_REDIS;
 const RedisConstructor = useMockRedis ? MockRedis : Redis;
 
 // Export either a real Redis instance, or Mocked.
-const redisInstance = new RedisConstructor(redisUrl);
+const redisInstance = new RedisConstructor(port, host, {
+  password: process.env.REDIS_PASSWORD,
+});
+
+function createRedisClient() {
+  return new RedisConstructor(port, host, { password: process.env.REDIS_PASSWORD });
```

Comment:
Question: what happens if the server doesn't set a password?  Will this method of doing it work?  Have you tested with/without a password?

Reply:
Testing

Reply:
Ran the app with the empty `PASSWORD` and it seems to be working fine. I Have conducted some research on google, and it seems that Redis now supports empty string `PASSWORD` as a `null` value, and build the object only using `port` and `host`. Nevertheless, the constructor is now located inside the try/catch block, which would secure us from getting unexpected errors in any case.